samples/CoNLL-U/ug-ud-test.conllu

```diff
@@ -0,0 +1,13030 @@
+# sent_id = s1
```

Comment:
@lildude I'm guessing the size of these sample files will be an issue?

Reply:
Oh my word!! Yes, definitely way too big. Chrome is also struggling with this diff expanded.

Please replace the samples with smaller samples that give a good representation of the structure of the language whilst still being real-world samples  (as opposed to written specifically for Linguist or a "hello world" equivalent).

Reply:
@pchaigno @lildude can I simple cut the file at some length? what's the ideal size?

Reply:
> @pchaigno @lildude can I simple cut the file at some length? 

Looking at the content, it does appear there is a fair amount of repetition so you could indeed cut the file down to only be a single highly representative "stanza".

> what's the ideal size?

Less than about 20KB so it doesn't bump into GitHub.com's [diff limits](https://help.github.com/articles/limits-for-viewing-content-and-diffs-in-a-repository/) is normally a good place to aim for, but we can accept slightly larger if it's absolutely necessary. 

The Farsi/Persian sample (Google thinks that's the language 😉 )  will be larger due to the high number of multi-byte characters so switching to English or other predominantly single-byte charactered language will also bring down the size.